ui/packages/upload-button/src/UploadButton.svelte

```diff
@@ -74,7 +74,7 @@
 </script>
 
 <input
-	class="hidden-upload hidden"
+	class="hidden-upload"
```

Comment:
Could we change this to `hide` to be consistent with other parts pf the code?

```suggestion
	class="hide"
```

Reply:
Easy peasy - will do!

Reply:
Done